.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "vendors/wxWidgets"]
+	path = vendors/wxWidgets
+	url = https://github.com/audacity/wxWidgets.git
```

Comment:
I believe it would be a little better to put the relative link here, like this:
`url = ../wxWidgets.git`
This way the user won't be forced to using HTTPS but the connection method will be inherited from audacity repo.

Reply:
Is that correct?  I'd expect that that `url = ../wxWidgets.git` would be relative to the file system (so it'd access an existing cloned copy).  I know git _can_ have remotes that are just other folders on the file system, though I don't know if doing that as a relative path there would trigger that or what exactly.

Reply:
Yep, you can try it yourself. As I understand it git uses both the parent's and the relative link  to make the final one which then will be used for syncing.

Reply:
@krokoziabla I just apply your advice!